OoyalaSkinSampleApp/OoyalaSkinSampleApp/Players/DefaultSkinPlayerViewController.m

```diff
@@ -63,6 +68,16 @@ - (void)viewDidLoad {
                                            selector:@selector(notificationHandler:)
                                                name:nil
                                              object:self.skinController];
+  [[NSNotificationCenter defaultCenter] addObserver: self
+                                           selector:@selector(notificationHandler:)
+                                               name:nil
+                                             object:ooyalaPlayer];
+  
+  // In QA Mode , making textView visible
+  if(self.qaModeEnabled==YES){
```

Comment:
Nitpicking but can you add a space after the `if` and around the `==` operator?
`if (self.qaModeenabled == YES)`